.github/workflows/ci.yaml

```diff
@@ -0,0 +1,71 @@
+---
+name: Generator tests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '^1.13.1'
+
+    # TODO(noahdietz): fix all lint errors then enable this.
+    # - name: Install golangci-lint
+    #   run: curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin latest
+    # - name: Run golangci-lint
+    #   run: golangci-lint run ./...
+    - name: Check license
+      run: go run ./util/cmd/license
+  unit-tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '^1.13.1'
+    - run: go test ./...
+  bazel-build:
+    runs-on: ubuntu-latest
+    container: gcr.io/gapic-images/googleapis-bazel:20210105
+    # Dockerfile for this image: https://github.com/googleapis/googleapis-discovery/blob/master/Dockerfile
+    # If you update its version, please also update it below in
+    # 'Cache Bazel files' - unfortunately it cannot accept variables at this
+    # time.
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Cache Bazel files
+      id: cache-bazel
+      uses: actions/cache@v2
+      with:
+        path: ~/.cache/bazel
+        key: ${{ runner.os }}-bazel-20210105-${{ secrets.CACHE_VERSION }}
```

Comment:
Right now there is no secret so it probably uses an empty string. That's OK! If we ever find that the cache is broken and we need to invalidate it, we'll add a `CACHE_VERSION` secret (any random string) and it will start over. This is a semi-official hack since there is no way to force cache removal.

Alternatively, you can just remove this step altogether if you don't need caching.

Reply:
I just removed all of the caching stuff.